com.cubrid.common.ui/src/com/cubrid/common/ui/er/editor/ERSchemaToolBar.java

```diff
@@ -409,15 +410,13 @@ private String getDBLabel() {
 	private Composite createAutoLayoutComp() {
 		Composite comp = new Composite(this, SWT.NONE);
 		final GridLayout gdLayout = new GridLayout(1, false);
-		gdLayout.marginHeight = 1;
-		gdLayout.marginWidth = 1;
-		gdLayout.horizontalSpacing = 2;
-		gdLayout.verticalSpacing = 0;
+		final GridData gridData = new GridData(SWT.FILL, SWT.FILL, true, true);
+		
 		comp.setLayout(gdLayout);
-		comp.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, false));
+		comp.setLayoutData(gridData);
 
-		final Button autoLayoutButton = new Button(comp, SWT.PUSH | SWT.CENTER);
-		autoLayoutButton.setLayoutData(CommonUITool.createGridData(1, 1, 98, 20));
+		final Button autoLayoutButton = new Button(comp, SWT.PUSH | SWT.CENTER | SWT.FILL);
```

Comment:
I think this issue is related with OS's font.
In my mac, "Auto Layout" button is printed as below.
![before](https://cloud.githubusercontent.com/assets/16603187/21173971/db18f2ec-c21d-11e6-8673-82bc505aa016.png)

How about reduce the font size?
I attached some codes example about reduce the font size. 
```java
		Font font = autoLayoutButton.getFont();
		String fontName = font.getFontData()[0].name;
		int fontSize = (int)font.getFontData()[0].height;
		// If `fontSize -1` couldn't show all characters, you have to minus more number as you need.
		autoLayoutButton.setFont(new Font(getDisplay(), fontName, fontSize - 1, SWT.NONE));
```

If you have better good idea about this, notice to me anytime. Thank you. 😃